packages/logger/src/context-aware-app-insights-logger-client.ts

```diff
@@ -18,6 +19,14 @@ export class ContextAwareAppInsightsLoggerClient extends BaseAppInsightsLoggerCl
         this.telemetryClient.commonProperties = {
             ...baseProperties,
         };
+
+        if (!this.rootLoggerClient.isSetup()) {
+            await this.rootLoggerClient.setup();
+        }
+    }
+
+    public isSetup(): boolean {
+        return this.telemetryClient !== undefined && this.rootLoggerClient.isSetup();
```

Comment:
Moving `private initialized: boolean` in a base class as `protected` and have only `public isSetup()` in a base class allow simplify implementation in inherited class by directly setting/getting `initialized` value directly